docs/bokeh/switcher.json

```
@@ -1,15 +1,9 @@
 [
   {
     "name": "latest",
-    "url": "https://docs.bokeh.org/en/latest/"
-  },
-  {
-    "version": "3.4.1",
-    "url": "https://docs.bokeh.org/en/3.4.1/"
-  },
-  {
-    "version": "3.4.0",
-    "url": "https://docs.bokeh.org/en/3.4.0/"
+    "version": "3.4.2",
+    "url": "https://docs.bokeh.org/en/latest/",
+    "preferred": true
   },
   {
     "version": "3.3.0",
```

```suggestion
    "version": "3.3.4",
```

The last minor version in the 3.3 series is `3.3.4`. The same issue exists in branch 3.5. I don't know if it is worth to update this line.